packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -86,13 +86,9 @@ const Card = (props) => (
               {props.href && props.textLink && (
                 <ReadMoreContainer>
                   <ReadMore>
-                    {props.clickable ? (
-                      props.textLink
-                    ) : (
-                      <GatsbyLink href={props.href} noUnderline>
-                        {props.textLink}
-                      </GatsbyLink>
-                    )}
+                    <GatsbyLink href={props.href} noUnderline>
+                      {props.textLink}
+                    </GatsbyLink>
```

Comment:
You should not do HTML that nests links, they are illegal since HTML 5

`<a href="foo">  ....   <a href="foo">  ...  </a>  ... </a>`